### ISSUE TITLE
Base Template HTML

### ISSUE THREAD

**Issue by elliotframe** [Team-2D/GameScoreCentral#13]
- search bar
- nav bar

**Comment by elliotframe** [comment]
@halchester how much work have you done on this so far, I have made a nav bar in the process of messing about with bootstrap but don't want to waste effort on finishing it if you've already done this.

**Comment by halchester** [comment]
I've only getting started last night since there was no assignees in the issue so I took it. I'm only getting started with the base layout. But if u're finished with the navbar, feel free to assign it to yourself and work on it, I haven't pushed any changes so far.

**Comment by halchester** [comment]
For future reference, it's better if you assign the issue to yourself first, before working on it.

**Comment by elliotframe** [comment]
@halchester That's fair enough since it hadn't been touched yet tbf, I was busy over the weekend so only just started this in the past couple hours to learn how bootstrap works but I think I've got some decent progress on it, hence the lack of assignment lol. I'm gonna see if I can get this mostly done today.

**Comment by halchester** [comment]
No problem dude! Great stuffs, thanks! So the bootstrap base html file will have the necessary bootstrap CDN imports and all the rest of the HTML templates will extend from it. That way, we don't have to import bootstrap CDN in every HTML file, just in one base html file. That means all of `frontend` tagged tickets will need to wait on this (to extend the base html file), just a heads up @elliotframe

**Comment by elliotframe** [comment]
This still needs some refining so it adapts to screen resizing etc but seems usable for developing other templates. Not entirely sure if the search works but it's passing inputs through to the url at least.
